timemachine/testsystems/dhfr.py

```diff
@@ -20,3 +20,14 @@ def setup_dhfr():
     host_fns, host_masses = openmm_deserializer.deserialize_system(host_system, cutoff=1.0)
 
     return host_fns, host_masses, host_coords, box
+
+
+def get_dhfr_system():
+    with resources.path("timemachine.testsystems.data", "5dfr_solv_equil.pdb") as pdb_path:
+        host_pdb = app.PDBFile(str(pdb_path))
+
+    protein_ff = app.ForceField("amber99sbildn.xml", "tip3p.xml")
+    host_system = protein_ff.createSystem(
+        host_pdb.topology, nonbondedMethod=app.NoCutoff, constraints=None, rigidWater=False
+    )
+    return host_system
```

Comment:
Could `setup_dhfr` call this to avoid repetition?

Reply:
yes, willfix - got a bit too zealous on the copy and pasting

Reply:
ugh - this is not quite straightforward, because the `setup` method references things like `host_pdb`, etc. which are not in the `host_system`

I don't want to make this function too complicated, so i'm going to settle for duplication of code here.